src/Features/ERDDAP/Superlatives/index.tsx

```diff
@@ -0,0 +1,170 @@
+/**
+ * Superlatives, or what platforms experiencing the most extreme conditions right now
+ */
+import * as React from "react"
```

Comment:
Codacy found an issue: ['React' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6227039)